app/web/platform/nativeLink.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+
+export function getReactNativeWebView(): typeof window.ReactNativeWebView {
```

Comment:
Does React Native not have specific types exported? Not a big deal just seems more readable and that's how I usually see it.

Reply:
I don't think there are TypeScript types for `window.ReactNativeWebView`. I've searched the [repo](https://github.com/react-native-webview/react-native-webview) and couldn't find anything... I am not entirely sure how to do this nicer than what I have here, open to suggestions!